### PR TITLE
feat(store): add standalone features

### PR DESCRIPTION
**Pull request by arturovt**

### CODE REVIEW

File: packages/form-plugin/src/form.module.ts [hunk 1]
```diff
@@ -1,13 +1,12 @@
...
-import { NGXS_PLUGINS } from '@ngxs/store';
+import { NGXS_PLUGINS, NgxsStoreFeature, withNgxsPlugin } from '@ngxs/store';
+
 import { NgxsFormPlugin } from './form.plugin';
-import { ReactiveFormsModule } from '@angular/forms';
-import { FormDirective } from './directive';
+import { NgxsFormDirective } from './directive';
 
 @NgModule({
-  imports: [ReactiveFormsModule],
```
Comment (Carniatto) [inline packages/form-plugin/src/form.module.ts:8]: Should this import be moved to the directive `imports`?

**Comment by kuncevic** [comment]
great work @arturovt, looking forward that feature to became available at least in preview to be able to play around with that, as spoken to @Carniatto

**Comment by kuncevic** [comment]
great work @arturovt, looking forward that feature to became available at least in preview to be able to play around with that, as spoken to @Carniatto  //cc @markwhitfeld

**Comment by markwhitfeld** [comment]
@arturovt The failing bundlemon check is not about bundlemon limits, but rather some artifacts that are missing:
image

**Pull request merged**